pkg/accounts/adaptor/repository/dummy.ts

```diff
@@ -111,4 +115,32 @@ export class InMemoryAccountFollowRepository
     this.data.delete(follow);
     return Result.ok(undefined);
   }
+
+  async fetchOrderedFollowers(
+    accountID: ID<AccountID>,
+    limit: number,
+  ): Promise<Result.Result<Error, AccountFollow[]>> {
+    return Result.ok(
+      [...this.data]
+        .sort((a, b) => {
+          return a.getCreatedAt().getTime() - b.getCreatedAt().getTime();
+        })
+        .filter((f) => f.getTargetID() === accountID)
```

Comment:
```suggestion
        .filter((f) => f.getTargetID() === accountID)
        .sort((a, b) => {
          return a.getCreatedAt().getTime() - b.getCreatedAt().getTime();
        })
```
平均計算量的には, フィルタしてからソートしたほうがよいかと